test/switch_test.dart

```diff
@@ -49,16 +51,12 @@ void main() {
         test('forwards events', () async {
           outer.add(first.stream);
           await Future(() {});
-          first
-            ..add(1)
-            ..add(2);
+          first..add(1)..add(2);
```

Comment:
It looks like your SDK has an older version of dart format. If you update to the latest dev SDK the formatting should match CI.